src/services/authService.js

```diff
@@ -0,0 +1,42 @@
+import { apiUrl } from '../config.json';
+import { post, get } from './httpService';
+
+const loginEndpoint = apiUrl + '/login';
```

Comment:
should we have these endpoints in a constants file. Also should probably be in all caps like LOGIN_ENDPOINT. Then we can import them here and into the tests. That would make it that if we ever had to change the endpoint we wouldn't have to change it in 2 places

Reply:
There's already a `rest-constants` file, I can add it to that. I do feel like it's a waste though if we don't add all the endpoints and refactor the `Switch` component to use that file instead. In fact, I'd go one step further and propose that the whole routing be refactored. I feel like having those routes and their components in a separate file and exported as an array may help in the future if we need to mess with something like breadcrumbs, for example. What are your thoughts?

Reply:
I can see having a js file that holds a map of path to component for the switch but I'd say the url paths for the switch are different from the rest endpoint constants.

Reply:
That is true. It's mere coincidence that they end up matching. Anyway, we'll refactor if we need to.